x-pack/plugins/actions/server/lib/action_executor.test.ts

```diff
@@ -1137,6 +1140,7 @@ test('throws an error when params is invalid', async () => {
     status: 'error',
     retry: false,
     message: `error validating action params: [param1]: expected value of type [string] but got [undefined]`,
+    errorSource: TaskErrorSource.FRAMEWORK,
   });
 });
```

Comment:
In the test below `throws an error when passing isESOCanEncrypt with value of false`, can we update it to test for the error source inside the error that's thrown? I think there are few other tests that could be updated this way as well.

Reply:
This is good idea.
I wrapped those errors with try-catch and marked them as FRAMEWORK explicitly where they are thrown.
Modified the tests as well. Thanks.